local_build_dotnet_core_example.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
```

Comment:
Please remove this script.

Reply:
Agree. I will soon remove it. As an alternative, could I put these information into a *.md file as I find it useful for newbie like myself to learn how to do local build?

Reply:
Please see https://github.com/GoogleContainerTools/distroless/pull/516#discussion_r435952762.